src/client/App.css

```diff
@@ -122,36 +143,37 @@ h2 {
   grid-area: email;
 }
 
-#registrationForm .usernameLabel {
+.usernameLabel {
   grid-area: usernameLabel;
 }
-#registrationForm #username {
+#username {
   grid-area: username;
 }
 
-#registrationForm .passwordLabel {
+.passwordLabel {
   grid-area: passwordLabel;
 }
-#registrationForm #password {
+#password {
   grid-area: password;
   font-weight: 100;
 }
 
 #registrationForm #email,
-#registrationForm #username,
-#registrationForm #password {
+#username,
+#password {
   background: rgb(150,203,174);
   background: linear-gradient(153deg, rgba(150,203,174,0.500437675070028) 0%, rgba(142,197,209,0.500437675070028) 50%, rgba(233,171,125,0.5032387955182073) 100%);
   border: none;
+  height: 25px;
 }
 
-#registrationForm .submit {
-  grid-area: submitR;
+.submit {
+  grid-area: submit;
   margin-top: 20px;
+  margin-bottom: 10px;
+  height: 25px;
   background: #ffffff;
   border: none;
-  margin-bottom: 10px;
-  height: 30px;
   box-shadow: -2px -2px 5px #8ec5d18f, 
               2px 2px 5px #e9ab7d8f
```

Comment:
I didnt know you could format shadows like this, does it create a 50/50 gradient?

Reply:
Sort of, yes. The buttons have blue on the left and the top and orange on the right and the bottom which seems to be the default for 2 colours